lib/smart_answer_flows/locales/en/marriage-abroad.yml

```diff
@@ -144,6 +144,8 @@ en-GB:
         embassies_data: |
           +[data_partial:overseas_passports_embassies:overseas_passports_embassies]
 # payment methods
+        pay_by_master_or_visa: |
```

Comment:
I think I'd prefer this to be named `pay_by_mastercard_or_visa`. While `master` may imply `mastercard` in the context of `master_or_visa` my preference would always be for explicitness.